### ISSUE TITLE
Max pixel text contents too close

### ISSUE THREAD

**Issue by philips** [philips/supernote-obsidian-plugin#38]
First bug: At 200px on my iPhone the text contents have no padding so when two pages are adjacent the words run into each other.

![image](https://github.com/user-attachments/assets/caf28b11-87a2-4595-977a-e0873dcd475f)

@james-xli can you take a look at this?

**Comment by james-xli** [comment]
Yes, thanks! I see the issue. Will try to get to this in the next evening or two if I can, before I have to travel for the holidays.
